go.mod

```diff
@@ -1,48 +1,48 @@
 module github.com/getsops/sops/v3
 
-go 1.23.0
+go 1.24.0
```

Comment:
@getsops/maintainers same question as in the previous versions of the PR (the latest was #1950).